mobile/lib/modules/memories/ui/memory_card.dart

```diff
@@ -63,18 +63,7 @@ class MemoryCard extends StatelessWidget {
           LayoutBuilder(
             builder: (context, constraints) {
               // Determine the fit using the aspect ratio
-              BoxFit fit = BoxFit.fitWidth;
```

Comment:
I think this is the only line you need to change. Box fit cover looks better for memories if ter aspect ratios are nearly close enough.